cmd/tools/cmq/repair.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"github.com/urfave/cli"
+	"os"
+	"strings"
+	"time"
+)
+
+func repair(c *cli.Context) error {
+	if c.NArg() < 2 {
+		fmt.Printf("dest-uuid and cg-uuid not specified\n")
+		return nil
+	}
+
+	mc, err := newMetadataClient(getOpts(cliContext))
```

Comment:
Probably should enforce consistency All.

Reply:
+1

Reply:
+1 .. presuming that would force Cassandra to replicate the delete to all nodes. Downside is the operation might take longer, which I presume is okay?

Reply:
Perhaps best to use a default of "All", but leave it 'override-able' with the "-cons' flag to cmq:
```
        mc, err := newMetadataClient(getOpts(cliContext))

        mc.consistency = gocql.All // default to using 'All' consistency
        if cliContext.IsSet("consistency") { // override consistency, if specified
                mc.consistency = gocql.ParseConsistency(cliContext.String("consistency"))
        }
```

Reply:
Just set `gocql.All` for now. I don't anticipate a need to `repair` with lower consistency level.

Reply:
Never-mind added flag as suggested.